<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="robot_address" default="localhost"/>

    <group ns="tj2">
        <node name="constants_setter" pkg="tj2_networktables" type="constants_setter.py" output="screen" required="false">
            <param name="nt_host" value="$(arg robot_address)"/>
        </node>
        <node name="constants_relay" pkg="tj2_networktables" type="constants_relay.py" output="screen" required="false">
            <param name="nt_host" value="$(arg robot_address)"/>
        </node>
    </group>
</launch>
